src/views/month/single-event.php

```diff
@@ -193,6 +193,6 @@
 ?>
 
 <div id="tribe-events-event-<?php echo esc_attr( $event_id ); ?>" class="<?php tribe_events_event_classes() ?>" data-tribejson='<?php echo esc_attr( tribe_events_template_data( $post ) ); ?>'>
-	<h3 class="tribe-events-month-event-title"><a href="<?php echo esc_url( $link ) ?>" class="url"><?php echo $title ?></a></h3>
+	<h3 class="tribe-events-month-event-title"><span><a href="<?php echo esc_url( $link ) ?>" class="url"><?php echo $title ?></a></span></h3>
```

Comment:
...or maybe I can remove the span and apply the line-through to the <h3> tag instead ;p